week2/week_2_step_0/test/widget_test.dart

```diff
@@ -0,0 +1,21 @@
+// This is a basic Flutter widget test.
```

Comment:
Want to kill this comment and replace it with something more appropriate?

Also, Apache license headers =)